exercises/concept/captains-log/.docs/instructions.md

```diff
@@ -47,7 +47,12 @@ In the captain's log, stardates are usually rounded to a single decimal place.
 
 Implement the `format_stardate/1` function that will take a floating point number and return a string with the number rounded to a single decimal place.
 
+While there are valid approaches to this solution that might use an Elixir module, the implementation we are testing for expects an implementation using an Erlang function.
```

Comment:
This is well suited for an analyzer comment, but is a bit long for here, and the motivation for doing that is not for testing, it's for learning to use Erlang, so we shouldn't mention tests.
 
Like I suggested, "The implementation should use an Erlang function." or something like that would be best.

The short sentence can also go to task 3. I saw your comment on the issue, but we have been talking about changing the analyzer check [over here](https://github.com/exercism/elixir-analyzer/issues/340) to check for an Erlang function, so the requirement should be explicit.